openff/qcsubmit/common_structures.py

```diff
@@ -400,6 +415,44 @@ def to_string(self) -> str:
         Cavity {{{cavity_str}}}"""
         return pcm_string
 
+    def add_keywords(self, keyword_data: dict) -> dict:
+        keyword_data["pcm"] = True
+        keyword_data["pcm__input"] = self.to_string()
+        return keyword_data
+
+
+class DDXSettings(_BaseSolvent):
+    """
+    A simple settings class for the ddx solvent model to be used with Psi4
+    """
+
+    ddx_model: Literal["pcm", "cosmo"] = Field(
+        "pcm", description="The solvation model to use."
+    )
+    ddx_radii_scaling: PositiveFloat = Field(
+        1.1,
+        description="The scaling factor for the cavity spheres this also depends on the radii set chosen.",
+    )
+    ddx_radii_set: Literal["uff", "bondi"] = Field(
+        "uff", description="The atomic radii set to use."
+    )
+    ddx_solvent_epsilon: Optional[float] = Field(
+        None,
+        description="The dielectric constant of the solvent, if not specified the solvent type should be set.",
+    )
+    ddx_solvent: str = Field(
+        "water",
+        description="The name of the ddx supported solvent which should used, the epsilon value will be determined from `pyddx.data.solvent_epsilon`. Note that this value is ignored if the `ddx_solvent_epsilon` is provided.",
```

Comment:
not blocking, very minor
```suggestion
        description="The name of the ddx supported solvent which should be used, the epsilon value will be determined from `pyddx.data.solvent_epsilon`. Note that this value is ignored if the `ddx_solvent_epsilon` is provided.",
```